### ISSUE TITLE
[Screen Reader-Intelligent Support Experience-Feedback]: JAWS is announcing as "Attachment" while navigating using cursor mode over the button controls of 'Feedback'.

### ISSUE THREAD

**Issue by Amit8527510735** [microsoft/BotFramework-WebChat#3700]
Note:
Issue is observed with JAWS through out Iris chat window while navigating in cursor mode.

User Impact:
Screen reader users who navigate using JAWS cursor mode will be confused if JAWS is narrating as "Attachment" while navigating over the button controls of 'Feedback' instead of reading as button.

Product name, version, and build: Intelligent Support Experience/Chrome v.84
OS : Windows 10 2004 (OS Build 19041.450)
Browser: Chrome 84.0.4147.135
URL: https://servicesuat.microsoft.com/#/time/timeentry
Screen Readers: JAWS 2020.2004.13.400

Does this repro with other AT? (If applicable): Yes, JAWS

AT version and build (if applicable): JAWS 2020.2004.13.400

Prerequisite (if any): NA

Repro Steps:
Step 1: Open the above URL in Chrome browser with valid credentials and turn on JAWS.
Step 2: Navigate to 'Bot' button in header section and press Enter
Step 3: Iris dialog will open
Step 4: Enter 'Feedback' in type your message edit field and press enter to send the message.
Step 5: Enable JAWS cursor mode and navigate over the button controls of 'Feedback' using arrow keys.
Step 6: Verify the announcement of JAWS screen reader while navigating over the button controls of 'Feedback' in cursor mode.

Screenshots:
MAS 4 2 1_Button reading as Attachment
MAS 4 2 1_Time Transfer
MAS 4 2 1_Credit Request
![100317182-e91bea80-2fe1-11eb-91ea-34630491693d](https://user-images.githubusercontent.com/53263029/106737520-c985eb80-663c-11eb-8020-348b56d78d57.png)
![100317177-e7eabd80-2fe1-11eb-9e9b-28910f565cb4](https://user-images.githubusercontent.com/53263029/106737525-cab71880-663c-11eb-8ee9-8da622a5fffa.png)
![100317172-e5886380-2fe1-11eb-81ca-0b567c70e831](https://user-images.githubusercontent.com/53263029/106737529-cb4faf00-663c-11eb-8a12-4e84c5d9ff59.png)

Actual:
JAWS is announcing as "Attachment" while navigating using cursor mode over the button controls of 'Feedback'.
Ex: JAWS is narrating as 'i dont like something attachment'

Expected:
JAWS should narrate the correct role and description when focus comes to the buttons present in Feedback message card while navigating using cursor mode.
Ex: JAWS should narrate as 'i dont like something button'

Other instances where this issue repros:

Credit Request
Navigation: Start Over -> Bot Response -> Yes -> C.6908020210 -> Credit request screen
Time Transfer
Navigation: Start Over -> Bot response -> C.6908035600 -> Bot Response -> C.6908035600.001000 -> Time Transfer screen will be opened
Please refer attachment(MAS 4.2.1_Credit Request.png & MAS 4.2.1_Time Transfer.png)
WCAG Reference Link:
https://www.w3.org/WAI/WCAG21/Understanding/

**Comment by Amit8527510735** [comment]
#A11yMAS;#A11ySev2;#Accessibility;#WCAG4.2.1;#HCL;#HCL_BotFramework_WebChat;#Linked:BUG6353921;#LinkedBug

**Comment by Amit8527510735** [comment]
@amal-khalaf : Access granted on your alias. Closed bug https://github.com/microsoft/BotFramework-WebChat/issues/3619 due to SLA and created new bug for the same.

**Comment by amal-khalaf** [comment]
@Amit8527510735
The described issue regarding JAWS announcing Attachement for any selected control is due the Adaptive Card UI elements does not have the proper **aria-roledescription** added to it.
what WebChat do is it set **aria-roledescription='attachement'** on the parent ui element which will contain the rendered AC component. which what JAWS will be falling back to when there is no **aria-roledescription** specified on the active UI element.
so we considered this  as bug on the AC team and external to webchat.

**Comment by Amit8527510735** [comment]
@amal-khalaf  So as per your comment seems this bug is external to Adaptive Card Team?

**Comment by amal-khalaf** [comment]
@Amit8527510735  yes - could you transfer this issue over to the AC team?
